packages/styles/src/carbon-stories/Notification/Notification-story.js

```diff
@@ -27,6 +27,11 @@ const notificationProps = () => ({
   role: text('ARIA role (role)', 'alert'),
   title: text('Title (title)', 'Notification title'),
   subtitle: text('Subtitle (subtitle)', 'Subtitle text goes here.'),
```

Comment:
Can you remove this `subtitle` field since you are adding a new `subtitle` field?